test/new-e2e/tests/agent-platform/package-signing/package_signing_test.go

```diff
@@ -92,6 +92,26 @@ func TestPackageSigningComponent(t *testing.T) {
 }
 
 func (is *packageSigningTestSuite) TestPackageSigning() {
+	// Install the signing keys
+	if is.osName == "ubuntu" || is.osName == "debian" {
+		is.Env().RemoteHost.MustExecute("sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y apt-transport-https curl gnupg")
+		aptUsrShareKeyring := "/usr/share/keyrings/datadog-archive-keyring.gpg"
+		is.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo touch %s && sudo chmod a+r %s", aptUsrShareKeyring, aptUsrShareKeyring))
+		keys := []string{"DATADOG_APT_KEY_CURRENT.public", "DATADOG_APT_KEY_C0962C7D.public", "DATADOG_APT_KEY_F14F620E.public", "DATADOG_APT_KEY_382E94DE.public"}
+		for _, key := range keys {
+			is.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo curl --retry 5 -o \"/tmp/%s\" \"https://keys.datadoghq.com/%s\"", key, key))
+			is.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo cat \"/tmp/%s\" | sudo gpg --import --batch --no-default-keyring --keyring \"%s\"", key, aptUsrShareKeyring))
+		}
+		aptTrustedDKeyring := "/etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg"
+		is.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo cp %s %s", aptUsrShareKeyring, aptTrustedDKeyring))
```

Comment:
```suggestion
		aptTrustedKeyring := "/etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg"
		is.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo cp %s %s", aptUsrShareKeyring, aptTrustedKeyring))
```
Fixes a typo. As discussed, I think this step is not necessary, but we can keep it to be safe